validator/sequences/broken_config/config_pointset.attr

```diff
@@ -6,5 +6,5 @@
   "subType" : "config",
   "category" : "commands",
```

Comment:
I think these should be added to .gitattributes to ignore the diffs so it's clear to the reviewer!

Reply:
What files need to be included in this directory? Without any redaction, all the files will change with every run except for the `sequence.md`? Should we only include `sequence.md` and ignore the rest?